test/extensions/filters/http/ext_authz/ext_authz_integration_test.cc

```diff
@@ -813,13 +813,18 @@ TEST_P(ExtAuthzGrpcIntegrationTest, GoogleAsyncClientCreation) {
   initializeConfig();
   setDownstreamProtocol(Http::CodecClient::Type::HTTP2);
   HttpIntegrationTest::initialize();
-  initiateClientConnection(4, Headers{}, Headers{});
 
-  waitForExtAuthzRequest(expectedCheckRequest(Http::CodecClient::Type::HTTP2));
+  int expected_grpc_client_count = 0;
```

Comment:
per my earlier comment, I think this the number of creation-calls, not the current number of clients, right? That's why I suggested `expected_grpc_client_creation_count`. 

The reason I'm inferring this is the number of creation calls is that it is being read from a counter. If it was the actual number of active clients, then I think that would need to be read from a gauge.

Reply:
Yes you are right. Done